README.md

```diff
@@ -1 +1,588 @@
-# batching-blog
\ No newline at end of file
+# Batched updates blog
```

Comment:
Reactive Updates without VDOM

Asynchronous Rendering for Web Components

Efficient DOM Updates without VDOM

Efficient DOM Updates for Web Components